go.mod

```diff
@@ -12,9 +12,12 @@ require (
 	github.com/tidwall/match v1.0.1 // indirect
 	github.com/tidwall/pretty v0.0.0-20190325153808-1166b9ac2b65 // indirect
 	go.uber.org/atomic v1.5.1
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58
 	stathat.com/c/consistent v1.0.0
 )
 
 replace stathat.com/c/consistent v1.0.0 => github.com/stathat/consistent v1.0.0
 
+replace github.com/json-iterator/go => github.com/xujianhai666/go v1.1.10-0.20200118093701-ae604587eee1
```

Comment:
remove personal pkg.

Reply:
I should check the json-iterator has accept my work on this project, then I will remove this

Reply:
this pr looks good to me, I will remove this personal pkg first, and merge this PR to the master as the initial code of the admin.

Reply:
if remove replace, the code will crash @ShannonDing

Reply:
the function "func (iter *Iterator) NextToken() byte" was not export in json-iterator/go, shall we request a PR for it?

Reply:
how about "func (iter *Iterator) WhatIsNext() ValueType" ?